set.mm

```diff
@@ -22273,15 +22273,39 @@ primitive connectives (no defined terms) on the right-hand side.
      ~ mo2 and ~ mo4 .  (Contributed by NM, 8-Mar-1995.) $)
   df-mo $a |- ( E* x ph <-> ( E. x ph -> E! x ph ) ) $.
 
+  ${
+    $d x y ph $.  $d y ps $.  $d y ch $.
+    eubidv.1 $e |- ( ph -> ( ps <-> ch ) ) $.
+    $( Formula-building rule for unique existential quantifier (deduction
+       rule).  (Contributed by NM, 9-Jul-1994.)  Remove dependency on ~ ax-6 ,
+       ~ ax-7 , ~ ax-12 by adapting proof of ~ eubid .  (Revised by BJ,
+       26-Sep-2022.) $)
+    eubidv $p |- ( ph -> ( E! x ps <-> E! x ch ) ) $=
+      ( vy weq wb wal wex weu bibi1d albidv exbidv df-eu 3bitr4g ) ABDFGZHZDIZF
+      JCQHZDIZFJBDKCDKASUAFARTDABCQELMNBDFOCDFOP $.
+  $}
+
+  ${
+    $d x y ph $.
+    mobidv.1 $e |- ( ph -> ( ps <-> ch ) ) $.
+    $( Formula-building rule for "at most one" quantifier (deduction rule).
```

Comment:
Suggest a consistent spelling of at-most-one (see line 22924 for comparison)

Reply:
I agree. I began replacing `"at most one"` with `at-most-one` in the previous PR since in my opinion this is an improper use of quotation marks. This is obviously not finished. I'm going to update this one in this PR.